concepts/src/concept_csv_export.py

```diff
@@ -22,10 +22,11 @@
 # Globals -- modified only during initialization
 VERBOSE = False
 DOCKER = False
+DOCKER_CONTAINER = "openmrs-sdk-mysql"
 VERSION = "2.3"
 LOCALES = ["en"]
 DEFAULT_LOCALE = "en"
-ENCODING = "UTF-8"
+ENCODING = "unicode_escape"
```

Comment:
Not sure exactly what this does, but UTF-8 wasn't working for me, and I found this via Google.

Reply:
Interesting @mogoodrich .  You'll notice that @brandones originally had 'latin-1' for this, and changed it to 'UTF-8' on review here:  https://github.com/PIH/iniz-exporters/pull/3.  Not sure why it is failing now, but I never tried running it before...

Reply:
Brandon has a Mac, correct?  My wild guess is that it was some OS difference...

Reply:
(the underlying script just runs a command line and attempts to parse the results)

Reply:
I run Ubuntu and was using the Dockerized MySQL that the OpenMRS SDK sets up.

Reply:
Strange, I was doing the same, the only difference was I set up the dockerized container myself....